content/operate/kubernetes/security/manage-rec-credentials.md

```diff
@@ -64,24 +64,12 @@ curl -k --request POST \
 
 3. From outside the pod, update the REC credential secret.
 
-3a. Save the existing username to a text file.
-
-```sh
-echo -n "<current_username>" > username 
-```
-
-3b. Save the new password to a text file.
-
-```sh
-echo -n "<new_password>" > password
-```
-
-3c. Update the REC credential secret.
-
 ```sh
 kubectl create secret generic <cluster_secret_name> \
-  --from-file=./username \
-  --from-file=./password --dry-run \
+  --save-config \
+  --dry-run=client \
+  --from-literal=username=<username> \
+  --from-literal=password=<password> \
```

Comment:
Consider using `<new-password>` placeholder. Previously we had this emphasized via the now-removed 3b section.